CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # New Relic Ruby Agent Release Notes
 
+## Upcoming Release
+
+This upcoming release of the agent adds instrumentation for Active Support caching operations.
+
+- **Add Active Support Instrumentation**
+
+  Active Support instruments caching-related operations. This change includes the instrumentation in the agent. A new segment is created when one of the operations is called. The key, store, and other attributes will be included on the segment as params. [PR#1742](https://github.com/newrelic/newrelic-ruby-agent/pull/1742)
```

Comment:
> This change includes the instrumentation in the agent.

I'm not sure what this sentence means, really. Could we reword or remove it?

Reply:
There's some ambiguity here in the whole paragraph given that Active Support is what's providing a high level caching interface for a Rails or other type of Ruby app AND what's providing pub/sub notifications for events related to the caching. Here's an attempt at rewrite:

```
Instrumentation is now automatically provided for all [Active Support caching](https://guides.rubyonrails.org/caching_with_rails.html) operations. Whenever a caching operation is performed, a New Relic segment will be created that contains timing information as well as parameters for the cache key, store, and other relevant data. [PR#1742](https://github.com/newrelic/newrelic-ruby-agent/pull/1742)
```

> Instrumentation is now automatically provided for all [Active Support caching](https://guides.rubyonrails.org/caching_with_rails.html) operations. Whenever a caching operation is performed, a New Relic segment will be created that contains timing information as well as parameters for the cache key, store, and other relevant data. [PR#1742](https://github.com/newrelic/newrelic-ruby-agent/pull/1742)

Reply:
Great points! Thanks for the rewrite, James. Incorporated in: c5db1c26942912fcbe69ecc284567b95003ebf1c